code/practice/3.py

```
@@ -0,0 +1,25 @@
+# - exceptの順番は重要（具体的な例外 → その他の例外）
+
+# ここにコードを書く
+try:
+    input1 = int(input("数字1を入力してくだい: "))
+    input2 = int(input("数字2を入力してください: "))
+    result = input1 / input2
+except ZeroDivisionError:
+    print("0では割れません")
+except ValueError:
+    print("数字を入力してください")
+except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)